src/Resources/contao/dca/tl_content.php

```diff
@@ -716,9 +716,27 @@
 		(
 			'label'                   => &$GLOBALS['TL_LANG']['tl_content']['articleAlias'],
 			'exclude'                 => true,
-			'inputType'               => 'select',
-			'options_callback'        => array('tl_content', 'getArticleAlias'),
-			'eval'                    => array('mandatory'=>true, 'chosen'=>true, 'submitOnChange'=>true, 'tl_class'=>'w50 wizard'),
+			'inputType'               => 'picker',
+			//'options_callback'        => array('tl_content', 'getArticleAlias'),
+			'eval'                    => array(
+				'mandatory'    => true,
+				'tl_class'     => 'clr',
+
+				'foreignTable' => 'tl_article', // Foreign table for widget record fetching. @todo get from picker somehow?
+				'context'      => 'article', // Context for picker
+				'labelField'   => 'title', // Field for the label
+				'fieldType'    => 'radio', // Field type
+
+				// Custom record label callback (icon is added anyway)
+				'label_callback' => function (array $row) {
+				    return $row['title'] . ' (ID: ' . $row['id'] . ')';
+				},
+
+				// Multiple sortable selection
+//				'fieldType'    => 'checkbox',
+//				'sortable'     => true,
+            ),
+			// @todo – how to handle wizard?
```

Comment:
What does the wizard do in that case? I mean why is a wizard needed at all?

Reply:
Well previously it was a select menu field, so when you have selected the record you were able to edit it in popup by clicking the edit icon. Dunno if we should drop or adjust it here.

Reply:
imho that's not needed, you can add it in the callback if you like, can't you?

Reply:
Dropped it in b5e3f2d. I left the method in the class for BC reasons.